src/QMCApp/qmcapp.cpp

```diff
@@ -46,7 +46,7 @@ int main(int argc, char** argv)
   using namespace qmcplusplus;
 #ifdef HAVE_MPI
   mpi3::environment env(argc, argv);
-  OHMMS::Controller->initialize(env);
+  OHMMS::Controller = new Communicate(env.world());
```

Comment:
Memory leak. First allocated https://github.com/QMCPACK/qmcpack/blob/f1bfdccbd4abd1d57892c00dbb570ecbf2949370/src/Message/Communicate.cpp#L35

Reply:
the memory leak is there anyway because the global controller is initialized with `new` anyway. The correct way would be to use a `std::unique_ptr` but fixing the leak was not the idea here.